common/logging.go

```diff
@@ -82,56 +86,143 @@ func (l ConsoleLogger) IsLogLevel(level LogLevel) bool {
 	return l.LogLevel >= level
 }
 
+// Error logs 'Error' message.
 func (l ConsoleLogger) Error(format string, args ...interface{}) {
 	if l.LogLevel >= LogLevelError {
 		prefix := "[ERROR] "
 		l.output(os.Stdout, prefix, format, args...)
 	}
 }
 
+// Warning logs 'warning' message.
 func (l ConsoleLogger) Warning(format string, args ...interface{}) {
 	if l.LogLevel >= LogLevelWarning {
 		prefix := "[WARNING] "
 		l.output(os.Stdout, prefix, format, args...)
 	}
 }
 
+// Notice logs 'Notice' message.
```

Comment:
Should be // Note logs notice messages.
No need to put 'Notice' into quotes.  Same for other comment similar.
(Error, Warning, Debug, Trace)
Sometimes variable names are put into ticks
https://github.com/unidoc/unipdf/wiki/UniPDF-Developer-Guide#12-doc-comments
but not usually function names